.travis.yml

```diff
@@ -35,7 +35,7 @@ install:
 
 before_script:
   - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/usr/local/cuda/lib64:$CONDA_DIR/lib
-  - export PATH=$CONDA_DIR/bin:$PATH
+  - export PATH=$CONDA_DIR/bin:/home/travis/build/yahoo/CaffeOnSpark/spark-1.6.0-bin-hadoop2.6/bin:$PATH
```

Comment:
Can we remove this change? You have a similar statement in https://github.com/yahoo/CaffeOnSpark/blob/python_api_test/scripts/travis/travis_install.sh#L17-L18